gtp2ogs.js

```diff
@@ -305,6 +344,16 @@ class Game {
             this.state = gamedata;
             check_for_move();
         });
+        this.socket.on('game/' + game_id + '/clock', (clock) => {
```

Comment:
This socket subscription can't live here, it needs to live with the other `socket.on` calls in the `Connection` instance as `loadState` gets called whenever a move is needing to be generated